src/modules/chat_highlight_blacklist_keywords/index.js

```diff
@@ -184,6 +186,55 @@ class ChatHighlightBlacklistKeywordsModule {
             defaultValue: false,
             description: 'Automatically hide pinned highlights after 1 minute'
         });
+
+        settings.add({
+            id: 'soundHighlight',
+            name: 'Play Sound On Highlight',
+            defaultValue: false,
+            description: 'On highlight you hear a sound'
+        });
+
+        this.username = twitch.getChatServiceClient().configuration.username.toLowerCase();
+        this.soundHighlight = false;
```

Comment:
why are we storing this as a property of the class? why not just check the settings?

Reply:
Updated.